src/nvapi.cpp

```diff
@@ -381,6 +382,8 @@ extern "C" {
             return NvidiaDeviceNotFound(n);
         }
 
+        SetNgxDebugOptions();
```

Comment:
Since NGX is 64-bit only but we are producing both `nvapi64.dll` and 32-bit `nvapi.dll`, I'd surround this with some kind of `#if defined(_WIN64) … #endif` to avoid polluting `WoW64Node` branches of registry.